runtime/queries/json/highlights.scm

```diff
@@ -1,9 +1,20 @@
+[
+  (true)
+  (false)
+] @constant.builtin.boolean
+(null) @constant.builtin
+(number) @constant.numeric
```

Comment:
Maybe `constant.numeric.integer`? I know it's not necessarily correct, but if someone defines `constant.numeric.integer` and `constant.numeric.float` but not `constant.numeric` then this won't get highlighted.

Reply:
Though I guess we should just recommend styling `constant.numeric`